content/documentation/install-uninstall.md

```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
+    Open Terminal and type these lines in, hitting enter after each:
 
-    Go to the Downloads folder to make sure the Prince zip file is there. The Prince folder should have a number listed in its title. Make sure XX.X in Word or Notes document matches this number by deleting and replacing with the correct version numbers. The line may end in macos or macosx. Make sure the it matches exactly what the program looks like in your Downloads folder and delete the "x" if necessary.
+    ```text
+    cd Downloads/prince
+    sudo ./install.sh
+    ```
 
-    Once updated, paste the two lines together in Terminal. You will be prompted to enter your computer password. Press enter. You should receive a message that PrinceXML will be installed in the `/usr/local` directory. Press enter again. If successful you will see a message in the Terminal saying "installation complete."
+    You will be prompted to enter your computer password. Press enter. You should receive a message that PrinceXML will be installed in the `/usr/local` directory. Press enter again. If successful you will see a message in the Terminal saying "installation complete." Or, if you get a "no such file or directory" message after the first line, the file you dowloaded is either not in your Downloads folder, or is not named "prince". Correct as necessary and try again.
 
-    When complete, type `cd` into the Terminal to return to your home directory.
+    When complete, type `cd` into the Terminal to return to your {{< q-def "home directory">}}.
 
     ```text
     cd
     ```
 
-5. The **Quire CLI** ({{< q-def "command-line interface" >}}) is the core of Quire. It is used for creating, previewing, and outputting projects. You can learn more about the Quire CLI in the [*Quire CLI Commands*](/documentation/quire-cli/) chapter of our guide. Copy and paste the following line into your Terminal to download the Quire CLI to your computer from GitHub.
+6. **Quire** is operated through a {{< q-def "command-line interface" >}} (CLI) that enables you to create, preview and output publications using Terminal with commands like `quire new`, `quire preview` and `quire site`. You can learn more in the [*Quire Commands*](/documentation/quire-cli/) chapter of our guide. Copy and paste the following line into Terminal to install Quire:
 
     ```text
-    git clone https://github.com/gettypubs/quire-cli.git
+    npm install --global @thegetty/quire-cli
     ```
 
-    You may be asked to enter your GitHub username and password. When complete, install the CLI by copying and pasting the following commands into your Terminal and pressing enter. The first resets user permissions for your local directory, the second installs Quire.
+    The Quire installation process may take several minutes, during which time there will be considerable messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR. These likely indicate a failed installation.
```

Comment:
Is this still true? I didn't get as much messaging as the first time and it went pretty quick.

Reply:
Changed this to "a minute or two" and took out "considerable"